fixtures/sample-project-content/src/sample-collection/sample-component-fragment/fragment.json

```diff
@@ -3,5 +3,6 @@
   "htmlPath": "sample-fragment.html",
   "cssPath": "sample-fragment.css",
   "jsPath": "sample-fragment.js",
+  "fragmentConfigurationPath": "fragment-configuration.json",
```

Comment:
@victorg1991 please can you check if this configuration is being read? (Try placing config/html/css/js in different directories, etc etc)

Reply:
no